types/react/v15/jsx-dev-runtime.d.ts

```diff
@@ -1,2 +1,33 @@
 // Expose `JSX` namespace in `global` namespace
-import "./";
+import * as React from "./";
```

Comment:
I don’t understand why the `JSX` namespace isn’t exported here as well as for some other previous React versions.

Reply:
scope JSX is a new thing. Adding it was a big effort so I kept it contained to current React versions. Adding stuff to older types is more risky since less people use them.